includes/Core/Util/Plugin_Status.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Util\Plugin_Status
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2025 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+/**
+ * Plugin_Status class.
+ *
+ * @since n.e.x.t
+ */
+class Plugin_Status {
+
+	/**
+	 * Plugin installed identifier.
+	 *
+	 * @since n.e.x.t
+	 */
+	const PLUGIN_STATUS_INSTALLED = 'installed';
+
+	/**
+	 * Plugin not active identifier.
+	 *
+	 * @since n.e.x.t
+	 */
+	const PLUGIN_STATUS_ACTIVE = 'active';
+
+	/**
+	 * Plugin not installed identifier.
+	 *
+	 * @since n.e.x.t
+	 */
+	const PLUGIN_STATUS_NOT_INSTALLED = 'not-installed';
+
+	/**
+	 * The plugin path of the plugin being checked.
+	 *
+	 * @var string The plugin path of the plugin being checked.
+	 *
+	 * @since n.e.x.t
+	 */
+	public static $plugin_path;
+
+	/**
+	 * Helper method to retrieve plugin installation/activation status.
+	 *
+	 * @param string $plugin_path The plugin path.
+	 * @param string $plugin_url  The plugin URL.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string The status of the plugin.
+	 */
+	public static function get_plugin_status( $plugin_path = '', $plugin_url = '' ) {
+		static::$plugin_path = $plugin_path;
```

Comment:
This is not how it should be done. IB has a point to add the `get_plugin_file` method which was added intentionally because plugins might be installed in folders with arbitrary names. We need to implement that method and use it instead of this static variable.